core/src/main/java/org/mule/runtime/core/el/DataWeaveExpressionLanguageAdaptor.java

```diff
@@ -53,6 +50,8 @@
   public static final String ATTRIBUTES = "attributes";
   public static final String ERROR = "error";
   public static final String VARIABLES = "variables";
+  public static final String PROPERTY = "property";
+  public static final String PARAM = "param";
```

Comment:
if we go with `parameters` rather than `param` (which is consistent with `variables`), then instead of `property` it should be `properties` as well.

Could you confirm?